.gitignore

```diff
@@ -65,3 +65,4 @@ target/
 pyuvdata/data/
 pyuvdata/data/test/
 pyuvdata/GIT_INFO
+uvdata/data/test/*
```

Comment:
Not sure what I was doing before, but I'm not getting anything there anymore. Could be because I was running a subset of the tests. Anyway, I'll remove this line.